src/legacy/core_plugins/timelion/index.js

```diff
@@ -34,6 +34,9 @@ export default function (kibana) {
         ui: Joi.object({
           enabled: Joi.boolean().default(false),
         }).default(),
+        graphiteUrls: Joi.array().items(
+          Joi.string().uri({ scheme: ['http', 'https'] }),
+        ).default([]),
```

Comment:
What if we used `https://www.hostedgraphite.com/UID/ACCESS_KEY/graphite` as the default here? That way this change doesn't impact those which are using the default Graphite URL. We're primarily concerned with arbitrary user supplied URLs here.

Reply:
We can use that, but we still receive an error when using the `.graphite()`  function that the URL is not a valid graphite URL.
I don't think someone is really using this URL as it is,  as it is here as a "placeholder" more than a real URL. They still need to change the UID and ACCESS_KEY in that URL.

Reply:
@kobelb what are your thoughts on that?

Reply:
I agree they won't be able to use the placeholder URL, but I think it could serve as a useful template or guide for users who are trying to configure their whitelist. It tells them that a valid graphite URL should have both a `UID` and `ACCESS_KEY`.

Reply:
If the default isn't usable, I'm indifferent to us using it as the default, thanks for the clarification @markov00.